src/dbt/kipptaf/models/tableau/staging/stg_tableau__view_count_per_view.sql

```diff
@@ -1 +1,10 @@
-select *, from {{ source("tableau", "src_tableau__view_count_per_view") }}
+select
+    *,
+    timestamp(
+        datetime(
+            timestamp(parse_timestamp('%m/%d/%Y %I:%M:%S %p', created_at)),
+            'America/New_York'
+        )
+    ) as created_at_local_new_york,
```

Comment:
```
datetime(parse_timestamp('%m/%d/%Y %I:%M:%S %p', created_at), '{{ var("local_timezone") }}') as created_at_local
```

Reply:
`parse_timestamp('%m/%d/%Y %I:%M:%S %p', created_at) as created_at_timestamp` for the downstream join. No need to convert tz for comparison